appveyor.yml

```diff
@@ -35,8 +35,7 @@ test_script:
 - C:\Python36-x64\python.exe -m pytest ..\python
 
 deploy_script:
-- echo head repo branch %APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH%
-- ps: if ($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH -ne "master") { return 0}
+- ps: if ($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH -ne "master") { Exit-AppVeyorBuild }
```

Comment:
Why is this needed? The `deploy_script` shouldn't be getting called on non-master branches?

Reply:
Unfortunately it is. Look at the build history without it.